test/SlowTests/Tests/TestsInheritanceTests.cs

```diff
@@ -89,7 +89,7 @@ where Filter(method)
                         select method;
 
             var array = types.ToArray();
-            const int numberToTolerate = 6452;
+            const int numberToTolerate = 6451;
```

Comment:
you didn't moved a test to use the `RavenFact` right? try to rebase

Reply:
Done